### PR TITLE
Improve privacy policy configuration

### PR DESCRIPTION
**Pull request by csillag**
This is an alternate solution for the same problem #1345 is trying to solve.

Following a suggestion from @lukaw3d 

- if `!REACT_APP_ENABLE_OASIS_MATOMO_ANALYTICS`: show our Privacy Policy link in place of Privacy Settings button
- then make Privacy Policy link customizable in env, so PontusX can link somewhere else

### CODE REVIEW

**Review by lukaw3d — APPROVED** [review]
You probably still want customRoutes from https://github.com/oasisprotocol/explorer/pull/1345

File: src/app/components/AnalyticsConsent/index.tsx [hunk 1]
```diff
@@ -93,8 +92,22 @@ const StyledPrivacyButton = styled(Button)(() => ({
...
 }))
 
+export const PrivacyPolicyFooterLink: FC = () => {
+  const { t } = useTranslation()
+  const theme = useTheme()
+  return (
+    <Link
+      href={process.env.REACT_APP_PRIVACY_POLICY}
+      target="_blank"
+      sx={{ color: theme.palette.layout.main }}
```
Comment (lukaw3d) [inline src/app/components/AnalyticsConsent/index.tsx:102]:
```suggestion
      color="inherit"
```

File: src/app/components/AnalyticsConsent/index.tsx [hunk 2]
```diff
@@ -93,8 +92,22 @@ const StyledPrivacyButton = styled(Button)(() => ({
   fontSize: 'inherit',
 }))
```
Comment (lukaw3d) [inline src/app/components/AnalyticsConsent/index.tsx:93]: | Current Privacy Settings | Current Privacy Policy |
| --- | --- |
| effective-space-sniffle-vq54j7774jfp94-1234 app github dev_mainnet_sapphire  | effective-space-sniffle-vq54j7774jfp94-1234 app github dev_ (2)  |

Lets add `fontWeight: 700` here

**Review by lukaw3d — APPROVED** [review]

File: src/app/components/AnalyticsConsent/index.tsx (hunk 2 above)
Comment (csillag) [reply src/app/components/AnalyticsConsent/index.tsx]: done

**Pull request merged**